jooby-filewatcher/src/main/java/org/jooby/filewatcher/FirstOfPathMatcher.java

```diff
@@ -0,0 +1,29 @@
+package org.jooby.filewatcher;
+
+import java.nio.file.Path;
+import java.nio.file.PathMatcher;
+import java.util.List;
+
+class FirstOfPathMatcher implements PathMatcher {
+
+  private List<PathMatcher> matchers;
+
+  FirstOfPathMatcher(final List<PathMatcher> matchers) {
+    this.matchers = matchers;
+  }
+
+  @Override
+  public boolean matches(final Path path) {
+    for (PathMatcher matcher : matchers) {
```

Comment:
As I saw you like to use Java 8 stream API, whouldn't it be better to write as matchers.stream().filter(...).findFirst()... ?